mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -410,6 +410,9 @@ class Word2Vec extends Serializable with Logging {
       i += 1
     }
 
+    require(word2VecMap.size > 0, "The word2vec map should not be empty. You may need to check " +
```

Comment:
This could be `nonEmpty`, but isn't this already determined by whether `vocabSize == 0`? I don't know this code well.